app/controllers/application_controller.rb

```diff
@@ -1,5 +1,11 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user ||= User.first
+  protect_from_forgery with: :exception
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
```

Comment:
- You have done well so far, but currently, I can use your application without authentication, this is because you have not updated your application controller to authenticate a user before accessing any of your controller actions. Please fix this 🙏 

`before_action :authenticate_user!`